### ISSUE TITLE
Missing psexec.exe in the Deployment folder?

### ISSUE THREAD

**Issue by SteveBurkettNZ** [cinglis-msft/UpdateComplianceConfigurationScript#2]
Hi Charles, noted you were looking in to another issue with the Update Compliance Configuration Script, so hoping you're the guy. 

Found that the [current download](https://www.microsoft.com/en-us/download/details.aspx?id=101086) doesn't have the psexec.exe file in the Deployment folder, so I'm getting a '..\Psexec.exe is not recognised as an internal or external command..' type error when you run the RunConfig.bat.

That file got missed out accidently?

**Comment by cinglis-msft** [comment]
Yes, looks like it. We're working on fixing a few things with the script. Thank you for bringing this to our attention

**Comment by SteveBurkettNZ** [comment]
No probs. BTW, because it's using PSExec.exe, the Attack Surface Reduction rule for [Block process creations originating from PSExec and WMI commands](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/attack-surface-reduction#block-process-creations-originating-from-psexec-and-wmi-commands) triggers. So any customers with locked down Windows 10 configs or running MDATP will likely have issues with the Update Compliance script as it stands. It's a little unusual to see PSExec being used like this.

**Comment by braddonegan** [comment]
Hello,

We ended up taking out the psexec line in the Deployment script and changing it to this:
`%PowershellHome%\powershell.exe -ExecutionPolicy Bypass -noexit -Command %powershellCommand%`

Didn't see a reason to have it run on all the clients for broad deployment.

Thanks!

**Comment by cinglis-msft** [comment]
Thanks for the update, we'll keep this in mind as we work on the next version. There are some changing requirements, hence the delay in the next version.
